units/other/lootboxes/lootboxbronze.lua

```diff
@@ -29,13 +29,13 @@ return {
 		idleautoheal = 10,
 		idletime = 1800,
 		levelground = false,
-		mass = 4000,
+		mass = 749,
 		health = 33500,
 		maxslope = 10,
 		speed = 0.03,
 		maxwaterdepth = 0,
 		metalmake = 20,
-		movementclass = "HOVER3",
```

Comment:
this is not the correct change
the footprint of this model is also incorrect
 the original movedef is correct
the footprint is wrong